ws-service/route.ts

```diff
@@ -118,7 +123,8 @@ const altpaymentRouteConnection = async (socket: Socket): Promise<void> => {
   const headersForwardedAddresses = socket.handshake.headers['x-forwarded-for'] as string
   const userIp = headersForwardedAddresses === undefined ? '' : headersForwardedAddresses.split(',')[0]
   if (userIp === '') {
-    throw new Error('Local IP not defined.')
+    console.error('Local IP not defined.')
```

Comment:
IMO this is a "so what" error message. What needs to happen for this to resolve itself? What would cause it in the first place?